README.md

```diff
@@ -52,6 +52,17 @@ require(['cytoscape', 'cytoscape-popper'], function( cytoscape, popper ){
 
 Plain HTML/JS has the extension registered for you automatically, because no `require()` is needed.
 
+```html
+<script src="https://unpkg.com/@popperjs/core@2/dist/umd/popper.min.js"></script>
+<script src="https://unpkg.com/cytoscape-popper@2.0.0/cytoscape-popper.js"></script>
+```
+
+For tippy:
+
+```html
+<script src="https://unpkg.com/tippy.js@6/dist/tippy-bundle.umd.min.js"></script>
+```
+
```

Comment:
Could suggesting specific CDN links make novice users expect that those are the only links that would work (only those specific versions etc.)?

Would the demo HTML files be a better place to comment on / document these links?

Reply:
Yeah that could definitely help, maybe we could just link to the demo file rather than the demo here?

Reply:
Good idea.  There are demo links at the top, but maybe they could be also linked in the relevant sections to make them more noticeable.